src/App.tsx

```diff
@@ -1,7 +1,41 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type Props = {};
```

Comment:
Props is an empty type, it can be removed and the component can be defined as `React.Component<{}, State>`.

Reply:
Oh, it's aproved already, no need to re-review :)